### DISCUSSION TITLE
Welcome to framework-system Discussions!

### DISCUSSION THREAD

**Discussion by JohnAZoidberg** [FrameworkComputer/framework-system#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zachfeldman** [comment]
Hi @JohnAZoidberg ! I am a @FrameworkComputer Marketplace Engineer interested in keeping my systems up to date with the latest firmware and in developing firmware and software with Rust. Very excited to see this tool make a public release and start seeing a community built around it!

I plan to follow developments, answer any questions I can from folks in the community, and maybe even make a PR or two into this specific repo as time goes on.

Keep up the good work building these tools! :star:
